### PR TITLE
Initial python example for GET endpoint

### PR DESCRIPTION
**Pull request by vladiibine**
The GET operation is illustrated for the GET hotel resource, and GET hotel batch endpoints.

### CODE REVIEW

**Comment by aalhour** [comment]
Hey @vladiibine, the PR looks good to me, I am just going to write notes to make the code even more noob-friendly, nothing major though, just reusing some concepts from one function you wrote as proposal for the other one.

File: python_requests/example.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    :param str partner_name: the partner's name for use in the API
+    :param str api_key: the partner's API key, for use in the API
+
+    :rtype: str
+    :return: A JWT token
+    """
+    authorization_raw_key = '{}:{}'.format(partner_name, api_key)
+
+    auth_response = requests.post(
+        'https://analytics.trustyou.com/connect/api/v1.0/auth',
```
Comment (aalhour) [inline python_requests/example.py]: Noob Friendly comment #1, can you offload the URLs into `url` variables defined before the `requests.post` method?

The signature of the function call will look like: `requests.post(url, ...)`.

File: python_requests/example.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        headers={
+            'Authorization': 'Basic {}'
+            .format(base64.b64encode(authorization_raw_key.encode('ascii')).decode('ascii')),
```
Comment (aalhour) [inline python_requests/example.py]: Can you offload this base64 expression to another variable defined under the `authorization_raw_key` above? The naming will make it easier for people to imagine the purpose of this expression even without knowing any Python.

For example: 
```
base64_encoded_key = base64.b64encode(authorization_raw_key.encode('ascii')).decode('ascii')
```

And then the `headers` dictionary would like the following example:

```
headers = {
    'Authorization': 'Basic {}'.format(base64_encoded_key),
    ...
}
```

**Comment by jeryini** [comment]
Looks good by me!

**Pull request merged**